lib/spack/spack/test/data/unparse/README.md

```diff
@@ -0,0 +1,6 @@
+# Test data for unparser
+
+These are test packages for testing Spack's unparser.
+
+They are named with `.txt` suffixes so that they're not considered
+proper source files by the various checkers used in Spack CI.
```

Comment:
This seems like a good workaround. Is there any documentation about spack checkers we might want to add this as a note to?

Reply:
Do I understand correctly that the content of these files can be regenerated at any time by calling `spack pkg source <package>`? Could we add a small note on that in this file so someone who wants to add an additional test package later gets how to do that?

Reply:
They're just copies of packages -- they need to remain static so their hashes don't change.

Reply:
Ok, that sounds reasonable!